src/app/ui/components/virtuoso.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactNode, useState } from 'react';
+
+import { Box } from 'leather-styles/jsx';
+
+import { useViewportMinWidth } from '@app/common/hooks/use-media-query';
+import { useOnResizeListener } from '@app/common/hooks/use-on-resize-listener';
+
+interface VirtuosoWrapper {
+  children: ReactNode;
+  hasFooter?: boolean;
+  isPopup?: boolean;
+}
+
+function vhToPixels(vh: string) {
+  const numericHeight = +vh.replace('vh', '');
```

Comment:
This is converting `vh` to pixels so that it's a numeric value and we can compute the height of the wrapper, taking into account offsets needed for headers + footers